internal/testutils/rust.go

```diff
@@ -118,8 +118,7 @@ func TrackRustCoverage(t *testing.T, src string) (env []string, target string) {
 	})
 
 	return []string{
-		"CARGO_INCREMENTAL=0",
-		"RUSTFLAGS=-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cinstrument-coverage",
```

Comment:
hum, I thought you wanted to do the opposite, keeping the initial env variables that are advise by upstream to get more accurate profiling?

I just want to ensure this is done on purpose :)

Reply:
It was done on purpose, indeed. These variables didn't seem to impact much in our coverage but test timing, but I can add them back and we see how it performs in the long time, wdyt?